app/models/ability.rb

```diff
@@ -84,6 +78,60 @@ def to_list
     end
   end
 
+  def manage(project_id, project_string_key)
```

Comment:
This looks great!

Reply:
That's a relief! Let me push a couple of minor changes in response to your feedback.